packages/mds-agency/request-handlers.ts

```diff
@@ -402,77 +430,71 @@ export const submitVehicleTelemetry = async (req: AgencyApiRequest, res: AgencyA
       } else {
         logger.info(name, 'no unique telemetry in', data.length, 'items')
         res.status(400).send({
-          error: 'invalid_data',
-          error_description: 'none of the provided data was unique',
-          result: 'no new valid telemetry submitted',
-          unique: 0
+          error: new ValidationError('none of the provided data was unique')
         })
       }
     } else {
       const body = `${JSON.stringify(req.body).substring(0, 128)} ...`
       const fails = `${JSON.stringify(failures).substring(0, 128)} ...`
       logger.info(name, 'no valid telemetry in', data.length, 'items:', body, 'failures:', fails)
       res.status(400).send({
-        error: 'invalid_data',
-        error_description: 'none of the provided data was valid',
-        result: 'no valid telemetry submitted',
-        failures
+        error: new ValidationError('none of the provided data was valid', { failures })
       })
     }
   } catch (err) {
     res.status(400).send({
-      error: 'invalid_data',
-      error_description: 'none of the provided data was valid',
-      result: 'no valid telemetry submitted',
-      failures: [`device_id ${data[0].device_id}: not found`]
+      error: new ValidationError('none of the provided data was valid.', {
```

Comment:
This should actually be a 500 response, probably with the error being a `new ServerError()`.

Reply:
I don't want to make it a `ServerError` object, since the spec says it should be a string.